lib/bots/versionbot.ts

```diff
@@ -354,7 +341,7 @@ export class VersionBot extends ProcBot {
         generateWaffleReference(pr.body);
 
         // Now look through all the commits in the PR. Do the same thing.
-        return this.githubCall({
+        return this.dispatchToEmitter(this.githubEmitterName, {
```

Comment:
Possibly, but the point was to remove an extraneous call that didn't need to be there. *shrug*